### PR TITLE
added react wrappers for green core icons arrowRight, arrowUpRightSqu…

### PR DESCRIPTION
**Pull request by GauriKudtarkar**
…are, arrowWallDown

### CODE REVIEW

File: libs/react/src/lib/icons/arrowUpRightSquare.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { createComponent } from '@lit/react'
+import { IconSquareArrowTopRight } from '@sebgroup/green-core/components/icon/icons/square-arrow-top-right'
+import { getScopedTagName } from '@sebgroup/green-core/scoping'
+
+export const ArrowUpRightSquare = createComponent({
```
Comment (vsjolander) [inline libs/react/src/lib/icons/arrowUpRightSquare.tsx:6]: Name should be `SquareArrowTopRight` to match with icon lib naming.

**Review by vsjolander — CHANGES_REQUESTED** [review]
One small thing. Otherwise great work!

File: libs/react/src/lib/icons/arrowUpRightSquare.tsx (hunk 1 above)
Comment (GauriKudtarkar) [reply libs/react/src/lib/icons/arrowUpRightSquare.tsx]: Updated file

**Comment by astrit** [comment]
@GauriKudtarkar thank you for the update, will add some other things along the icons and merge it today ✌️

**Comment by astrit** [comment]
@GauriKudtarkar will close this PR in favor of #1522 where your contribution is also included.

**Pull request closed without merging**